debug-on-kubernetes.md

```diff
@@ -0,0 +1,59 @@
+# Debug support on Kubernetes cluster

```

Comment:
Would love to see a short introduction here before jumping into the numbered sections. Even just one or two sentences giving a quick overview of what/why.

Reply:
Thanks, Nice suggestion.